src/main/com/cqfn/degitx/simulator/randomengine/GUID.kt

```diff
@@ -0,0 +1,15 @@
+package com.cqfn.degitx.simulator.randomengine
+
+import java.util.*
+import kotlin.streams.asSequence
+
+const val longSource = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+const val GUIDLength = 8L
+
+class GUID {
```

Comment:
please add comments

Reply:
@stepanvalyavskiy ,thank for catching that missing. Fixed.